PLCSystemLogic.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collection.Generic;
+
+namespace mujinplccs
+{
+    public class MujinSystem{
+        List<Location> locations;
```

Comment:
Is there a class called `Location`?

Reply:
We might need a `GetLocationByLocationIndex` for easier retrieving the info of the location (which might suggest a hashtable/ dictionary is needed if the locationIndex of the location does not relate to the index of the location list)

Reply:
why list?

Reply:
not decided yet,  a collection of Locations